.github/workflows/acceptance-tests-runner.yml

```diff
@@ -246,12 +250,7 @@ jobs:
             - 'internal/service/ldapconfiguration/*.go'
             - 'internal/service/ldapverify/*.go'  
           network:
-            - 'internal/service/networkcontainer/*.go'
```

Comment:
same with these removals

Reply:
aah sorry nice catch, these changes sneaked in from my previous branch, let me clean up
Thank you!!

Reply:
fixed, resolving comment, feel free to reopen